src/controllers/notification.controller.js

```diff
@@ -0,0 +1,33 @@
+import path from 'path';
+import { notificationServices } from '../services';
+
+const getAllNotifications = async (req, res) => {
+  const results = await notificationServices.getNotifications(req.user.id);
+  res.status(200).json({
+    Code: '200',
+    Message: 'All notifications retrieved successfully',
+    results,
+  });
+};
+
+// const NotificationAccess = async (req, res) => {
+//   res.status(200).json({
+//     code: 200,
+//     message: 'Access Granted',
+//     data: req.user,
+//   });
+// };
```

Comment:
why this comments

Reply:
Resolved. I mistakenly forgot to uncomment that while debugging.

Reply:
Resolved. I mistakenly forgot to uncomment that while debugging.